src/Chamilo/CourseBundle/Entity/CLp.php

```diff
@@ -266,20 +276,240 @@ class CLp
      */
     protected $accumulateScormTime;
 
+    /**
+     * @var Course
+     * @ORM\ManyToOne(
+     *     targetEntity="Chamilo\CoreBundle\Entity\Course",
+     *     inversedBy="learningPaths",
+     *     cascade={"persist", "remove"}
+     * )
+     * @ORM\JoinColumn(name="c_id", referencedColumnName="id")
+     */
+    protected $course;
```

Comment:
@jmontoyaa Can this association mapping for entities generate a change in the DB structure?

Reply:
Yes, It will create FKs in the database.

Reply:
Which should not create any problem, since only new Chamilo installations will get this new foreign key constraint (which is a security). Existing database will not be altered.

Reply:
...which leaves us with only the large changes to the core processes of Chamilo (even if only a change in form), which is a bit too risky to include in the code a few days before the release (we have to live with that with users support for months if there is something wrong).
The inclusion of this PR is thus postponed to after the 1.11.12 release.
I want to stress that this is not about bad code or anything (to the countrary). It is about the size of the changes and the corresponding risk.